tests/endtoend/test_dependency_isolation_functions.py

```diff
@@ -7,7 +7,7 @@
 from unittest.mock import patch
 
 from requests import Response
-from azure_functions_worker import testutils
+from tests.utils import testutils
 from azure_functions_worker.utils.common import is_envvar_true
 from azure_functions_worker.constants import PYAZURE_INTEGRATION_TEST
 
```

Comment:
Should we explore putting testing-related constants within `test.utils`?

Reply:
Good point. Added a new constants file under tests/utils